immutrack/src/components/login/Login.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { connect } from "react-redux";
+import { axiosWithAuth } from "../../utils/axiosWithAuth";
+import { Form, Field, withFormik } from "formik";
+import * as Yup from "yup";
+// import Header from '../base/Header'
+// import Footer from '../base/Footer'
+
+const Login = ({ user }) => {
+  return (
+    <div>
+      {/* <Header /> */}
+      <Form>
+        <Field
+          component="input"
+          type="text"
+          name="email"
+          autoComplete="false"
+        />
+        <Field
+          component="input"
+          type="password"
+          name="password"
+          autoComplete="false"
+        />
+        <button type="submit">submit</button>
+      </Form>
+      {/* <Footer /> */}
+    </div>
+  );
+};
+
+const HOCForm = withFormik({
+  mapPropsToValues({ email, password }) {
+    return {
+      email: email || "",
+      password: password || ""
+    };
+  },
+  validationSchema: Yup.object().shape({
+    // email: Yup.string().required(),
+    // password: Yup.string().required()
+  }),
+  handleSubmit(values, { resetForm, props }) {
+    axiosWithAuth()
+      .post(`/auth/login/${props.user}`, values)
+      .then(res => {
+        console.log(res);
+        // localStorage.setItem("token", res.data.token);
+        props.user === "parents"
+          ? props.history.push("/")
+          : props.history.push("/login");
+      })
+      .catch(err => {
+        console.log(err);
+        resetForm();
+        props.history.push("/user");
```

Comment:
Which component will have this '/user' path?

Reply:
None. Forgot to add/commit that change. This will be /login - a failed attempt will push to /login.